checkstyle-custom/src/test/java/io/vivy/checkstyle/custom/MethodCallParenShouldMatch.java

```diff
@@ -0,0 +1,24 @@
+package io.vivy.checkstyle.custom;
+
+import com.puppycrawl.tools.checkstyle.Checker;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class MethodCallParenShouldMatch {
+
+    GenericFunction genericFunction = new GenericFunction();
```

Comment:
can't find the class for it. Where is it? Should be named a bit different to show the provided functionality